### PR TITLE
[RM Ch03] Change openconfig link Issue #3151

### LINKED ISSUE #3151: [RM] Fix gates: https://openconfig.net/ certificate is incorrect
https://build.opnfv.org/ci/job/cntt-tox-rm/1682/console

`(chapters/chapter03: line 1804) broken    https://openconfig.net/ - HTTPSConnectionPool(host='openconfig.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)')))`

I would suggest to change the link as best option. Or to ignore it as worst option.

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #3151

### CODE REVIEW

**Review by collivier — APPROVED** [review]

**Comment by collivier** [comment]
@walterkozlowski could you have quickly a look as it breaks gates? the change seems trivial and safe and should be merged asap.

**Review by gkunz — APPROVED** [review]

File: doc/ref_model/chapters/chapter03.rst [hunk 1]
```diff
@@ -1806,7 +1806,7 @@ for their respective control functions. These APIs must have an ability to handl
 forwarding and control planes life cycle management will not be atomic. The offload functions that should be offered as
 services must have published and preferably standardized open APIs, but the application specific forwarding functions do
 not have to be open APIs since they will only communicate with the application tenant provided control functions.
-`P4 <https://p4.org/>`__ and `OpenConfig <https://openconfig.net/>`__ are examples of suitable languages and models,
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter03.rst:1809]: Is it clear why the linkcheck fails on this? https://openconfig.net/ opens in my browser nicely. Anyways I'm okay to change this to fix the builds, but would be nice to know.

**Review by CsatariGergely — APPROVED** [review]

File: doc/ref_model/chapters/chapter03.rst (hunk 1 above)
Comment (collivier) [reply doc/ref_model/chapters/chapter03.rst:1809]: @CsatariGergely the certificate is clearly invalid - I'm hoping your web browser is warning u about such issues.

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter03.rst:1809]: @CsatariGergely It is a certificate issue and, thus, for example, it is blocked at MSFT on any browser. I also didn't work on my personal MAc - Safari.

**Pull request merged**